### PR TITLE
Fix joint names in joint_limits.yaml file for dual arm example

### PR DESCRIPTION
**Pull request by bi3ri**
Hello,

the names in joint_limits.yaml file needed to be adjusted for the prefix left and right otherwise one would get the following error message. 

```bash
RTConnect'. Additional configuration parameters will be set when the planner is constructed.
[move_group-3] [ERROR] [1676219710.198921626] [moveit_trajectory_processing.time_optimal_trajectory_generation]: No acceleration limit was defined for joint right_panda_joint1! You have to define acceleration limits in the URDF or joint_limits.yaml
[move_group-3] [WARN] [1676219710.198946193] [moveit_ros.add_time_optimal_parameterization]:  Time parametrization for the solution path failed.
```

Versions:
ROS2 humble
moveit2 humble tag 2.7.0

cheers,
Johannes

### CODE REVIEW

File: doc/examples/dual_arms/CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 add_subdirectory(dual_arm_panda_moveit_config)
 
-install(DIRECTORY launch
+install(DIRECTORY
   DESTINATION share/${PROJECT_NAME}
 )
```
Comment (AndyZe) [inline doc/examples/dual_arms/CMakeLists.txt:5]: Thanks for noticing that this launch file was unused!

I believe you can delete this line completely now:

```suggestion
```

**Review by AndyZe — APPROVED** [review]

**Review by sjahr — APPROVED** [review]
@bi3ri  Thanks for addressing this issue! This looks good. 

Can you fix the CI failures? You can format the code by running [pre-commit run -a](https://moveit.ros.org/documentation/contributing/code/#pre-commit-formatting-checks)

**Comment by AndyZe** [comment]
Merging this b/c the "htmlproofer" points out a dead link on an unrelated MTC page. I'll fix that in a followup PR. Thanks @bi3ri

**Comment by bi3ri** [comment]
@AndyZe Happy that it got merged and thanks for you guidance!

**Pull request merged**